internal/core/src/mmap/Utils.h

```diff
@@ -124,6 +78,12 @@ WriteFieldData(File& file,
                 }
                 break;
             }
+            case DataType::VECTOR_SPARSE_FLOAT: {
+                // TODO(SPARSE): this is for mmap to write data to disk so that
+                // the file can be mmaped into memory.
+                throw std::runtime_error(
+                    "WriteFieldData for VECTOR_SPARSE_FLOAT not implemented");
```

Comment:
this means mmap is not supported for mmap yet?

Reply:
yes, no mmap support for sparse yet.